package.json

```diff
@@ -267,6 +267,8 @@
   },
   "resolutions": {
     "elliptic": "^6.5.3",
-    "nwmatcher": "^1.4.4"
+    "nwmatcher": "^1.4.4",
+    "http-proxy-agent": "^5.0.0",
```

Comment:
What package makes you need to add the resolutions?

Reply:
"vscode-proxy-agent": "^0.12.0" references v4 of http agent - and that's the latest version available of that package.